styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  font-family: "Arial", sans-serif, sans-serif;
+  background-color: #f4f4f4;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
You can use `dvh` as u unit. It does the same thing as `vh` but works better on mobie phones. Here you can read some more 
https://blog.stackademic.com/exploring-css-units-unleashing-the-power-of-dvh-over-vh-1ff61b9c87ae